ui/packages/consul-ui/app/controllers/dc/acls/policies/edit.js

```diff
@@ -32,8 +32,23 @@ export default class EditController extends Controller {
     );
   }
 
-  @action
-  delete(item) {
+  // Forwarders replacing route-action usage
+  @action onCreate(item, event) {
```

Comment:
```suggestion
  @action
  onCreate(event, item) {
```

two nit picks here it's not call to action but more of observation:
* having decorator on its own line is common format
* is there a reason you are passing event as a second arg? if there is no particular reason it would be more obvious in the future when we start using function on on modifers where event is always going to be first

Reply:
Fixed the decorator formatting.

Regarding (item, event) ordering
We are binding the item with onCreate when we are calling using fn in the corresponding template file:
`@onCreate={{fn this.onCreate item}}`

This will then come up as the first argument.

For consistency with how on modifiers work, I believe we will have to do refactoring at multiple places.